src/main/java/core/basesyntax/enums/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.enums;
+
+public enum Color {
+    RED,
+    GREEN,
+    BLUE,
+    YELLOW,
+    ORANGE
+    }
```

Comment:
The closing brace for the enum declaration should be on the same line as the last enum constant if it's the only thing on the line, or on a new line indented correctly if followed by a semicolon and methods or fields.